README.rst

```diff
@@ -845,6 +845,55 @@ wait_for_motion(timeout_secs=10, consecutive_frames=None, noise_threshold=None,
       to search for motion. White pixels select the area to search; black
       pixels the area to ignore.
 
+wait_until(callable_, timeout_secs=10, interval_secs=0)
+    Run a function or lambda until it succeeds, or until a timeout.
```

Comment:
Perhaps "Wait until a condition becomes True or timeout".  Talking about function "success" might suggest some interaction with exceptions, e.g. when `print("hello")` succeeds it returns a `False`ish value (`None`).